test-cases/tests/backup-restore/j05b-verify-3scale-backup-and-restore.md

```diff
@@ -78,13 +78,20 @@ Once verified, delete the throwaway Postgres
 oc delete -n redhat-rhoam-operator postgres/throw-away-postgres
 ```
 
-3. Run the backup and restore script
+3. **Non-STS** - Run the backup and restore script
 
 ```sh
 cd test/scripts/backup-restore
 ./j05-verify-3scale-postgres-backup-and-restore.sh | tee test-output.txt
 ```
 
+4. **STS** - Reach out to QE for the `osdCcsAdmin` credentials in order to run the backup and restore script
```

Comment:
Perhaps we should use the credentials that SRE would use? The osdsupport ones so that we are also verfying it has the aaccess needed?